server/server.c

```diff
@@ -13,71 +13,242 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <fcntl.h>
+#include <syslog.h>
+#include <errno.h>
+#include <signal.h>
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <netdb.h>
 #include <netinet/in.h>
 
+void gracefulShutdonw()
+{
+    remove("/var/tmp/aesdsocketdata");
```

Comment:
@sankalp123321  Is this function safe to do in signal handler?

Reply:
It's not, I changed the design in assignment 6